openapi.yaml

```diff
@@ -83,108 +121,85 @@ parameters:
     type: string
     required: true
     description: '"Bearer [token]" where [token] is your OAuth2 access token'
-  pageNumber:
-    name: page[number]
-    in: query
-    type: integer
-    required: false
-    description: Page number of results
-    default: 1
-    minimum: 1
-  pageSize:
-    name: page[size]
-    in: query
-    type: integer
-    required: false
-    description: Number of results to return
-    default: 25
-    maximum: 500
-    minimum: 1
 definitions:
-  Meta:
-    properties:
-      totalResults:
-        type: integer
-        description: Total number of results
-        example: 10
-      totalPage:
-        type: integer
-        description: Total number of pages
-        example: 10
-      currentPageNumber:
-        type: integer
-        description: Page number of the returned results
-        example: 1
-      currentPageSize:
-        type: integer
-        description: Number of results per page
-        example: 25
   SelfLink:
     properties:
       self:
         type: string
         format: url
         description: Self-link of current resource
-  PaginationLinks:
-    properties:
-      first:
-        type: string
-        format: url
-        description: The first page of data
-      last:
-        type: string
-        format: url
-        description: The last page of data
-      prev:
-        type: string
-        format: url
-        description: The previous page of data
-      next:
-        type: string
-        format: url
-        description: The next page of data
-  PetResource:
+  TermResource:
     properties:
       id:
         type: string
-        description: A unique ID of pet
+        description: A 6-digit unique term code
       type:
         type: string
-        enum: [pet]
+        enum: [term]
```

Comment:
Could you put quotes around the string in the list for consistency? I suppose we might want to change that in the skeleton too.

Reply:
Good point. I'm also not sure should we quote everything including descriptions in the swagger file. Looks at the example in [swagger site](https://swagger.io/docs/specification/adding-examples/), seems like they just quote the string very randomly as well...

Reply:
Right, it looks like it's actually more common to not quote the values in the `enum`. It probably makes sense to quote strings representing versions, like `2.1.2` since similar values like `2.1` would be incorrectly parsed as a float. Strings with characters that need to be escaped like `"` should also probably be quoted. Same with `$ref` links. Other than that, I don't think it's necessary.